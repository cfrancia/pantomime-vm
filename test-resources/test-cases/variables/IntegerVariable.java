/**
 * Created by cfrancia on 24/09/16.
 */
public class IntegerVariable {

    public static void main(String[] args) {
        int i = 2_048_000;
        println(i);
    }

    public static native void println(int val);

}
