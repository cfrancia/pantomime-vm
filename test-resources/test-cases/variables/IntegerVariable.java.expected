OUT: 2048000
